cal/schema.py

```diff
@@ -43,6 +42,7 @@ class User(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String, unique=True, nullable=False)
```

Comment:
Hm...maybe we should just make `name` the primary key? At the very least, we should probably add an index for `name`.

Possibly ditto w/ `Event`